tests/cupy_tests/core_tests/test_ndarray.py

```diff
@@ -466,3 +466,11 @@ def test_size_axis_too_small(self, xp):
     def test_size_zero_dim_array_with_axis(self, xp):
         x = testing.shaped_arange((), xp, numpy.int32)
         return xp.size(x, 0)
+
+
+@testing.gpu
+class TestPythonInterface(unittest.TestCase):
+
+    def test_bytes_tobytes(self):
+        x = testing.shaped_arange((3, 4, 5))
+        assert x.tobytes() == bytes(x)
```

Comment:
We would like to test that both numpy and cupy arrays yield the same values. I.e. compare the output of bytes(numpy_array) to bytes(cupy_array)

This should be done for an empty array, an scalar, and a several elements array.